### ISSUE TITLE
Allure integration - skipped steps broke the report

### ISSUE THREAD

**Issue by jalona** [Invictum/invictum-test#91]
Serenity-Cucumber + your allure integration

If an error occurred in the cucumber scenario and then there are steps - they are not marked as SKIPPED
And previous steps not write in allure

**Comment by Invictum** [comment]
@jalona 
I am unable to reproduce your issue. Could you provide a simple project where its occurred?

**Comment by jalona** [comment]
Cucumber scenario
```
# language: ru

@issue:HA-140

@master
Функционал: Name functional

  Сценарий: Name scenario
    Пусть step 1 - no exception
    И step 2 - no exception
    То step 3 - exception
    То step 4 - no exception
```

Cucumber step definition
```
public class MainPageSteps extends BaseStep {

    @Пусть("^step 1 - no exception$")
    public void step1() {
        // empty
    }

    @И("^step 2 - no exception")
    public void step2() {
        // empty
    }

    @И("^step 4 - no exception$")
    public void step4() {
        // empty
    }

    @И("^step 3 - exception$")
    public void step3() {
        throw new IllegalStateException("Example exception");
    }
}
```

Result allure test suite xml
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1489038200201" stop="1489038201811" version="1.5.2" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Name functional</name>
    <test-cases>
        <test-case start="1489038200335" stop="1489038200623" status="broken">
            <name>Name scenario</name>
            <failure>
                <message>IllegalStateException: Example exception</message>
                <stack-trace>java.lang.IllegalStateException: Example exception
	at ru.homecredit.at.sample.web.itportal.stepdefinitions.MainPageSteps.step3(MainPageSteps.java:28)
	at ✽.То step 3 - exception(One.feature:11)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="PC077469"/>
                <label name="thread" value="5200@PC077469.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="CucumberJVM"/>
        <label name="feature" value="Name functional"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
```
![image](https://cloud.githubusercontent.com/assets/7051715/23737531/c6c00a14-04a4-11e7-94f6-7c89afa3e0a3.png)

**Comment by Invictum** [comment]
Implemented. Will be available in 1.4 version.
